unit_of_time/__init__.py

```
@@ -11,26 +12,26 @@ def date_from_int(val, div=1):
     return mul * (val.year * 10000 + val.month * 100 + val.day)
 
 
 class TimeunitKindMeta(type):
-    kind_int = None
-    formatter = None
-    _pre_registered = []
-    _registered = None
-    _multiplier = None
+    kind_int: int = -1
+    formatter: str = ""
+    _pre_registered: list["TimeunitKindMeta"] = []
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)